src/connector/src/sink/iceberg/mod.rs

```diff
@@ -142,6 +143,15 @@ pub struct IcebergConfig {
     // We should try to find and use that as default commit retry num first.
     #[serde(default = "default_commit_retry_num")]
     pub commit_retry_num: u32,
+
+    /// Whether to enable iceberg compaction.
+    #[serde(default, deserialize_with = "deserialize_bool_from_string")]
+    pub enable_compaction: bool,
+
+    /// The interval of iceberg compaction
+    #[serde(default)]
+    #[serde_as(as = "Option<DisplayFromStr>")]
+    pub compaction_interval: Option<u64>,
```

Comment:
Explicitly show the unit and change the name to `compaction_interval_in_seconds`?